lib/index.js

```diff
@@ -37,12 +37,15 @@ module.exports = function (options) {
 			res.end = end;
 			res.end(chunk, encoding);
 
-			if (!req.route || !req.route.path) {
-				return;
-			}
-
+			let statTags = [...tags];
+			
+			const routePath = req.route && req.route.path ? req.route.path : '';
```

Comment:
I feel like you could place all the `route` related lines of code into a 

```javascript
if (req.route && req.route.path) {
  const route = baseUrl + replacePipeChar(req.route.path)
  statTags.push(`route:${route}`);
}
```

Reply:
Currently, my change defaults the `route` tag to the base URL if the `base_url` flag is true.  This would be beneficial for requests handled like this:
```
const STATIC_DIR = process.env.STATIC_DIR || "/content/static";
app.use("/static", express.static(STATIC_DIR));
```
Your suggested change would leave off the `route` tag for those requests, making it hard to aggregate metrics for them in Datadog.

Reply:
You're right, I've was too fast here 🙇 👍